### DISCUSSION TITLE
Creating new gtd files and integrating them in the import process

### DISCUSSION THREAD

**Discussion by Wall-AF** [NationalSecurityAgency/ghidra#6148] [General]
What's the process to create a new gtd file for say an older version of Visual Studio and ensure Ghidra picks this when importing an executable?

**Comment by Wall-AF** [comment]
What was the process used for creating  the current `.gtd` & `.fidb` files?

**Reply by ryanmkurtz** [reply]
This is not really my area, but I believe the FID files are generated with the GhidraScripts under the "FunctionID" folder in the Script Manager.

I think the .gtd files are created with the "Parse C Source" feature.
